app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -110,3 +110,4 @@ CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
 CONFIG_COMP_STUBS=y
```

Comment:
The git commit prefix is wrong, this patch applies to one Intel platform only. And what about other Intel SOF targets? Shouldn't we have same timer source for all Intel platforms?